lessons/07-html-forms/index.html

```diff
@@ -0,0 +1,43 @@
+<html>
+    <head>
+        <title>Create User</title>
+    </head>
+    <body>
+        <main>
+            <h1>Create user</h1><hr>
+            <form action="create-user.html" method="get">
+                <label for="email">Email : </label>
+                <input type="text" name="email" placeholder="email" required><br><br>
+
+                <label for="username">User name : </label>
+                <input type="text" name="username" placeholder="username" size="30" maxlength="12"><br><br>
+
+                <label for="password">Password : </label>
+                <input type="text" name="password" maxlength="8" placeholder="password" required><br><br>
+
+                <label for="bio"> Bio:</label><br>
+                <textarea name="bio" placeholder="Give a brief description of this user"></textarea><br><br>
```

Comment:
We need to reserve using `<br />` only for when we want a new line inside of a paragraph.

What we can do here instead is group the inputs + labels together with `<div>` tags 

```html
<div>
  <label for="username">User name : </label>
  <input type="text" name="username" placeholder="username" size="30" maxlength="12">
</div>

<div>
  <label>Password:</label>
  <input type="password" name="password" placeholder="password" size="30" maxlength="8" required/>
</div>
```